ci/docker/Dockerfile.build.ubuntu

```diff
@@ -137,17 +137,27 @@ RUN cd /usr/local && \
 # https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
 ARG BASE_IMAGE
 RUN export SHORT_CUDA_VERSION=${CUDA_VERSION%.*} && \
+    wget -O nvidia-ml.deb https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb && \
+    dpkg -i nvidia-ml.deb && \
```

Comment:
Why? You shouldn't need this

Reply:
This allows us to choose the right version

Reply:
My point is that the nvidia ml repository should already be available and you don't need to install it a second time? Prior to your change, there was no issue to run `apt-get install -y "libnvinfer-dev=6.0.1-1+cuda10.2`

Reply:
This method guarantees us to have the right nvinfer version no matter what happens to the container in the future

Reply:
Currently the container already contains the machine learning repo in `/etc/apt/sources.list.d/nvidia-ml.list`. Why would the container team decide to break their users by removing the machine learning repo? Is there any other reason for re-installing the already existing repository?

Reply:
I noticed the following issue with the container though I'm not sure about the impact https://gitlab.com/nvidia/container-images/cuda/-/issues/82